### PR TITLE
Add Clock.saver screensaver v0.5.0

### PR DESCRIPTION
**Pull request by witsch**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
https://github.com/soffes/Clock.saver/releases/tag/v0.5.0

> https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md
> The Cask is for an app that is unmaintained (no releases in the last year, or explicitly discontinued).

This hasn't had a release in three years.

**Comment by witsch** [comment]
There have been new commits this March, I'll ask the author if he's planning a release…

**Comment by witsch** [comment]
Wow, three mistakes in two words! 😳😄 I suppose that's what happens if you're mostly a user and only contribute the odd cask once every few weeks…  Thanks Vítor for clarifying!

PS: was it really (day before) yesterday?  If yes, congratulations! 🎉

**Comment by vitorgalvao** [comment]
> was it really only (day before) yesterday? In any case, congratulations!

It was. Thank you!

File: Casks/clocksaver.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'clocksaver' do
+  version '0.5.0'
```
Comment (soffes) [inline Casks/clocksaver.rb]: 0.5.1 is out now!

**Comment by witsch** [comment]
@commitay there's a new release out at https://github.com/soffes/Clock.saver/releases/tag/v0.5.1, do you want me to squish the commits into one for this pull request or is it okay like this?

**Comment by commitay** [comment]
This is fine as it is, GitHub lets us squash when merging.

Thanks @soffes and @witsch!

**Comment by witsch** [comment]
Thank you, and squash, of course, not squish… 😃

**Pull request merged**